platform/security/platform-security-core-api/src/main/java/ddf/security/SubjectUtils.java

```diff
@@ -156,10 +218,29 @@ public static String getName(Subject subject, String defaultName, boolean return
         return name;
     }
 
+    private static String getExtendedCertAttribute(X500Principal principal,
```

Comment:
I'll add a test for the case where the method can return null, but there is no legitimate way of getting the RDNs to be null, that is only there because of how that method is written. You literally cannot create a principal in such a way that it would be null because it is validated when creating the principal.